smartconnect/__init__.py

```diff
@@ -115,6 +116,25 @@ def __init__(self, *, api=None, username=None, password=None, use_language_code=
         self.logger = logging.getLogger(SmartClient.__name__)
         self.verify_ssl = smart_settings.SMART_SSL_VERIFY
 
+    def get_server_version(self):
```

Comment:
@jgoodheart is this API available now? Or is this placeholder code for when it is made available?

Reply:
its available but they just released an updated server build that has the new permissions in it. This was implemented for my initial testing but is not used yet